src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -66,27 +88,76 @@ function EnsureMember($group, $userOrGroup) {
 ## TODO: Consider creating a lock around this script because it is not concurrency safe
 
 if ($install) {
-    InstallAPI
+    InstallAPI $serviceName
+} else {
+    try {
+        $pingEndpoint = "https://localhost:$adminAPIPort"
+        Write-Verbose "Pinging Admin API at $pingEndpoint"
+        Invoke-WebRequest -UseDefaultCredentials -UseBasicParsing $pingEndpoint | Out-Null
+    } catch {
+        if ($_.Exception.Status -eq [System.Net.WebExceptionStatus]::ConnectFailure) {
+            $service = Get-Service -Name $serviceName -ErrorAction SilentlyContinue
+            if (!$service) {
+                throw "IIS Administration API is not installed"
+            }
+            if ($service.Status -eq [System.ServiceProcess.ServiceControllerStatus]::Running) {

```

Comment:
-eq => -ne  ?

Reply:
If the service is running and we can't find the service on the specified port, we send this error message.

This is as intended